src/ekore/anomalous_dimensions/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,160 @@
+"""
+The unpolarized time-like leading-order 
+(LO) Altarelli-Parisi splitting kernels.
+
+"""
+
+import numba as nb
+import numpy as np
+from eko import constants
+from ....harmonics import cache as c
+
+
+
+@nb.njit(cache=True)
+def gamma_qq(N, cache, is_singlet=None):
+    """Computes the LO quark-quark anomalous dimension.
+    Implements Eqn. (B.3) from :cite:`Mitov:2006wy`.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment 
+    cache : numpy.ndarray
+        Harmonic sum cache
+    is_singlet : boolean
+        True for singlet, False for non-singlet, None otherwise
+
+    Returns
+    -------
+    gamma_qq : complex
+        LO quark-quark anomalous dimension 
+        :math:`\gamma_{qq}^{(0)}(N)` 
+
+    """
+    s1 = c.get(c.S1, cache, N, is_singlet)
+    result = constants.CF * (-3.0 + (4.0 * s1) - 2.0 / (N * (N + 1.0)))
+    return result
+
+@nb.njit(cache=True)
+def gamma_qg(N):
+    """Computes the LO quark-gluon anomalous dimension.
+    Implements Eqn. (B.4) from :cite:`Mitov:2006wy` 
+    and Eqn. (A1) from :cite:`Gluck:1992zx`.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment 
+
+    Returns
+    -------
+    gamma_qg : complex
+        LO quark-gluon anomalous dimension 
+        :math:`\gamma_{qg}^{(0)}(N)` 
+
+    """
+    result = - (N**2 + N + 2.0) / (N * (N + 1.0) * (N + 2.0))
+    return result
+
+@nb.njit(cache=True)
+def gamma_gq(N, nf):
+    """Computes the LO gluon-quark anomalous dimension.
+    Implements Eqn. (B.5) from :cite:`Mitov:2006wy` 
+    and Eqn. (A1) from :cite:`Gluck:1992zx`.
+
+    Parameters
+    ----------
+    N : complex
+        Mellin moment 
+    nf : int
+        No. of active flavors 
+
+    Returns
+    -------
+    gamma_qg : complex
+        LO quark-gluon anomalous dimension 
+        :math:`\gamma_{gq}^{(0)}(N)`
+
+    """
+    result = (-4.0 * nf * constants.CF * (N**2 + N + 2.0) 
+    / (N * (N - 1.0) * (N + 1.0)))
+    return result
+
+@nb.njit(cache=True)
+def gamma_gg(N, nf, cache, is_singlet=None):
```

Comment:
```suggestion
def gamma_gg(N, nf, cache):
```

you know you are in the singlet case here - so no need for the argument (this is of course also true for all other functions)

Reply:
Yes, I know that, however in #202, I have modified the cache.get to take is_singlet as an argument as well. Since that function is called in gamma_qq and gamma_gg for the s1, I just set the is_singlet to None to ensure no missing arguments are there. 

Indeed I will have to rebase this branch, and that's why I have been trying to complete the #202 to have it fully working.

Reply:
> Yes, I know that, however in https://github.com/NNPDF/eko/pull/202, I have modified the cache.get to take is_singlet as an argument as well. Since that function is called in gamma_qq and gamma_gg for the s1, I just set the is_singlet to None to ensure no missing arguments are there.

mh? how you mean? yes, you need to pass the argument to `get`, but this does not mean it has to be an argument for `gg` - on the contrary, `gg` enforces the argument to be `True` (forever) - or am I missing something?

Reply:
> Indeed I will have to rebase this branch, and that's why I have been trying to complete the https://github.com/NNPDF/eko/pull/202 to have it fully working.

I don't expect #202 to be closed any time soon ... so you should keep #202 working (as it was) and gradually increase the code basis whenever you make a new step

Reply:
> mh? how you mean? yes, you need to pass the argument to `get`, but this does not mean it has to be an argument for `gg` - on the contrary, `gg` enforces the argument to be `True` (forever) - or am I missing something?

It does not enforce to to be True. It sets it to be None, and then when the gamma_singlet is defined, it is then set to True, as it should be. I did this as a way to keep the code consistent across all orders. A little more than needed, I agree, but cleaner imho.

Reply:
I still don't see this point (and I forgot to ask during the meeting) - @giacomomagni may I ask you for a comment?

Reply:
well, its just a matter of code consistency across perturbative orders. At LO, in principal, I could hard code the cache.get to have is_singlet = True but, this way, the code is consistent such that every splitting function that uses cache, needs an argument is_singlet, and this is specified in the __init__ file making it very consistent.

Reply:
Sorry I'm a bit confused... I need to check #202. But I believe a function called `gamma_gg` can only be singlet,
so the argument should not be need otherwise it's really confusing. Instead `is_singlet` should be enforced in all the harmonics called inside this function.